### PR TITLE
allow to set quality of image

### PR DESCRIPTION
**Pull request by halidovz**

### CODE REVIEW

File: capture-video-frame.js [hunk 1]
```diff
@@ -5,12 +5,13 @@
...
     }
 }(this, function () {
-    return function captureVideoFrame(video, format) {
+    return function captureVideoFrame(video, format, quality) {
         if (typeof video === 'string') {
             video = document.getElementById(video);
         }
 
         format = format || 'jpeg';
+        quality = quality || 1;
```
Comment (ilkkao) [inline capture-video-frame.js]: MDN says `0.92` is the default, can you use that?

Comment (halidovz) [reply capture-video-frame.js]: oh, yep, I've just noticed that too.

**Comment by ilkkao** [comment]
Thanks a lot!

**Comment by ilkkao** [comment]
Released capture-video-frame@0.1.3

**Pull request merged**